README.md

```diff
@@ -29,26 +29,36 @@ will need to support both published releases and [pre-release versions
 (#12)](https://github.com/openshift-kni/install-scripts/issues/12) of
 each of these.
 
-The scripts will:
+To ease installation, a [prepared ISO](https://github.com/openshift-kni/install-scripts/issues/20)
+can be used to install the "provioning host".  Using the prepared ISO addresses
+the following: 
 
 1. [Creates an admin user
    (#21)](https://github.com/openshift-kni/install-scripts/issues/21)
    with passwordless sudo on the provisioning host.
 1. Ensure the provisioning host has all [required software
-   installed](https://github.com/openshift-kni/install-scripts/blob/master/01_install_requirements.sh). This
-   script will also be used to [prepare an ISO image
-   (#20)](https://github.com/openshift-kni/install-scripts/issues/20)
-   to speed up this part of the installation process.
-1. [Validate any environment requirements
+   installed](https://github.com/openshift-kni/install-scripts/blob/master/01_install_requirements.sh). 
    (#22)](https://github.com/openshift-kni/install-scripts/issues/22) -
    the [bare metal IPI network
    requirements](https://github.com/openshift/installer/blob/master/docs/user/metal/install_ipi.md#network-requirements)
    are a good example of environment requirements.
```

Comment:
This sentence seems like it's been orphaned here - check out the rendered version

Reply:
right,

tightened up the note, adding some l3 headers  and added an intro to the list below